go/user/UserImpl.go

```diff
@@ -0,0 +1,29 @@
+package user
+
+import (
+	"typedb_driver/go_wrapper"
+)
+
+type UserImpl struct {
+	users UserManagerImpl
+	nativeObject typedb_driver.User
+}
+
+func NewUserImpl(user typedb_driver.User, users UserManagerImpl,) *UserImpl {
+	return &UserImpl{
+		users:        users,
+		nativeObject: user,
+	}
+}
+
+func (u *UserImpl) username() string {
```

Comment:
`Username` or `username`? Let's fix it everywhere if you put it here

Reply:
Yep - should be capital as it is public.